helm-chart/templates/deployment.yaml

```diff
@@ -63,12 +63,17 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           workingDir: /opt/unnamed-thingity-thing/comptest
           command:
+          {{- if .Values.adminUsers }}
```

Comment:
If there are no admin users to add, we can probably skip the initContainer entirely!